packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -0,0 +1,672 @@
+The SCIM server response is to return the group details, just as for a group retrieval request:
+
+    HTTP/1.1 200 OK
+    Date: Fri, 18 Oct 2019 07:11:00 GMT
+    Content-Type: text/json;charset=UTF-8
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "74094a55-c9ee-47ae-9fd4-9137deb43497",
+        "displayName": "Test SCIM2",
+        "members": [],
+        "meta": {
+            "created": "2019-10-18T07:02:44+00:00",
+            "lastModified": "2019-10-18T07:11:00+00:00",
+            "version": "3a0c8354ccfcaab56019859469078a0b"
+        }
+    }
+
+### Update specific group membership
+
+<ApiOperation method="patch" url="/groups/${groupId}" />
+
+To add users to a specific pushed group on the SCIM server, Okta requires:
+
+* The user must be a member of the group in Okta
+* The user has been added under the **Assignments** tab of the SCIM application inside the Okta Admin Console
+* The group is pushed under the **Push Groups** tab of the SCIM application inside the Okta Admin Console
+
+If these three requirements are met; Okta sends a PATCH request to add a user on the SCIM server. The request looks similar to the following:
+
+    PATCH /scim/v1/Groups/74094a55-c9ee-47ae-9fd4-9137deb43497 HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "members": [{
+            "value": "48e0a2da-0999-4f2c-87f4-80432cfe6617",
+            "display": "test.user@okta.local"
+        }]
+    }
+
+The SCIM server response is to return the group details, just as for a group retrieval request:
+
+    HTTP/1.1 200 OK
+    Date: Fri, 18 Oct 2019 07:11:01 GMT
+    Content-Type: text/json;charset=UTF-8
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "74094a55-c9ee-47ae-9fd4-9137deb43497",
+        "displayName": "Test SCIM2",
+        "members": [{
+            "value": "48e0a2da-0999-4f2c-87f4-80432cfe6617",
+            "display": "test.user@okta.local"
+        }],
+        "meta": {
+            "created": "2019-10-18T07:02:44+00:00",
+            "lastModified": "2019-10-18T07:11:01+00:00",
+            "version": "cca3a6ab18596cc450377eddcc4b5a58"
+        }
+    }
+
+When a user is removed from the group, Okta sends the `members` array with an `operations` element that has the value `delete`.
+
+### Deleting specific groups
+
+<ApiOperation method="delete" url="/groups/$[groupId}" />
+
+Okta administrators can remove pushed groups under the **Push Groups** tab of the SCIM application inside the Okta Admin Console, In the **Push Groups** tab, click **Active** then **Unlink pushed group**. In the dialog box that appears, Okta provides an option to delete the group on the SCIM server:
+
+![Unlink a pushed group in Okta Admin console](/img/scim11-unlink-group.png "Unlink a pushed group in Okta Admin Console")
+
+When the admin clicks on the **Unlink** button, Okta sends a DELETE request:
```

Comment:
remove "on", "the", and the word "button"

When the admin clicks **Unlink**, Okta sends a DELETE request: